src/features/auth/register-form/RegisterFormStyles.tsx

```diff
@@ -0,0 +1,100 @@
+import { useMediaQuery, useTheme } from "@mui/material";
+
+export const FormBoxStylesFunct = () => {
+
+  const theme = useTheme();
+  const isSmallScreen = useMediaQuery(theme.breakpoints.down("sm"));
+
+  const FormBoxStyles = {
+    width: isSmallScreen ? "250px" : "350px",
+    height: "680px",
+    margin: "auto",
+    padding: isSmallScreen ? "18px" : "40px",
+    backgroundColor: "white",
+    display: "flex",
+    flexDirection: "column",
+    alignItemns: "center",
+    justifyContent: "center",
+    borderRadius: "8px",
+    marginBottom: "16px",
+  };
+
+  return { FormBoxStyles };
```

Comment:
No lo deconstruyamos
```suggestion
  return formBoxStyles;
```